app/core/models.py

```diff
@@ -244,12 +257,22 @@ class ScoreBase(Timestamped):
     class Meta:
         abstract = True
 
+    def clean(self):
+        if self.score_type.level != self.REQUIRED_LEVEL:
+            raise ValueError(
+                "Score Type {} cannot be set on an {} score".format(
+                    self.score_type, self.REQUIRED_LEVEL
```

Comment:
```suggestion
                f"Score Type {self.score_type} cannot be set on an {self.REQUIRED_LEVEL} score"
```
Any reason to not use an `f-string` here?

Reply:
none